h5c/vic/src/vic-webapp/src/app/create-vch-wizard/networks/networks.scss

```diff
@@ -8,7 +8,14 @@
   width: 6em;
 }
 
-:host ::ng-deep clr-signpost .signpost-action {
-  margin-top: 0;
-  margin-bottom: 0;
+:host ::ng-deep clr-signpost {
+  .signpost-action {
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+}
+
+.bridge-network-signpost {
+  min-width: 600px;
+  max-width: 600px;
```

Comment:
Why is this necessary here to set min and max to the same? Can we not set the width only?

Reply:
Since the signpost is inside a col-xs-1, it is using the min-width which results in a thin and tall signpost popup due to the long text inside. Setting width to signpost max-width now.